modules/installation-operators-config.adoc

```diff
@@ -52,5 +57,43 @@ service-catalog-apiserver            4.4.0     True        False         False
 service-catalog-controller-manager   4.4.0     True        False         False      5m25s
 storage                              4.4.0     True        False         False      5m30s
 ----
+endif::ibm-z[]
+ifdef::ibm-z[]
+----
+$ oc get clusteroperators
+NAME                                       VERSION                                   AVAILABLE   PROGRESSING   DEGRADED   SINCE
+authentication                             4.3.0-0.nightly-s390x-2020-03-09-183623   True        False         False      40d
```

Comment:
Is it correct for the versions here to say 4.3 nightly?

Reply:
Yeah, `nightly` isn't acceptable. Do you think changing `4.3.0-0.nightly-s390x-2020-03-09-183623` to `4.3.0-s390x` is reasonable here?

Reply:
I would say that's at least better than nightly, but it would be good to find out what the actual output would be here for a 4.3 release.

Reply:
Changed.